index.php

```diff
@@ -0,0 +1,31 @@
+<?php
+header("Cache-Control: no-store, no-cache, must-revalidate");
+header("Cache-Control: post-check=0, pre-check=0", false);
+header("Pragma: no-cache");
+header("Last-Modified: " . gmdate("D, d M Y H:i:s") . " GMT");
+header("Expires: Sat, 26 Jul 1997 05:00:00 GMT");
+
+$config = include('config.php');
+date_default_timezone_set("US/Pacific");
```

Comment:
I assume this matches the timezone set on the `digitalservices.sfgov.org` server?

Reply:
i better check that!

Reply:
it matches _now_